<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="fon">
   <property name="geometry">
    <rect>
     <x>-3</x>
     <y>-3</y>
     <width>1000</width>
     <height>1000</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="fon2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>100</y>
     <width>371</width>
     <height>531</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="foto">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>120</y>
     <width>331</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="opis">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>370</y>
     <width>331</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="vihod">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>700</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="tekst">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>-3</y>
     <width>650</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_01">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>100</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_02">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>100</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_03">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>100</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_04">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>160</y>
     <width>100</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_05">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>100</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_06">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>100</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_07">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>330</y>
     <width>100</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_08">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>330</y>
     <width>100</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
